focus-ios/Blockzilla/uk.lproj/Localizable.strings

```diff
@@ -44,7 +44,7 @@
 "actionSheet.openIn" = "Відкрити в %@";
 
 /* Button to dismiss the 'Add Pass Failed' alert. */
-"AddPass.Error.Dismiss" = "Гаразд";
+"AddPass.Error.Dismiss" = "OK";
```

Comment:
@Delphine I was surprised about this change which seems like undoing a translated string...

Reply:
Oh I see! Some languages actually decide to use "OK" in some cases. I've checked across other strings and seems like this is expected for Ukrainian. Thanks for checking!

Reply:
Sounds good to me then! Thanks!